### PR TITLE
Phase2-gem27 Move GEM related geometry files in the style suggested by David

### PR DESCRIPTION
**Pull request by bsunanda**
Correct the scenario defining files as well

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+1

File: Geometry/HcalCommonData/python/testPhase0GeometryXML_cfi.py [hunk 1]
```diff
@@ -14,7 +12,7 @@
         'Geometry/CMSCommonData/data/muonBase.xml',
         'Geometry/CMSCommonData/data/cmsMuon.xml',
         'Geometry/CMSCommonData/data/mgnt.xml',
-        'Geometry/CMSCommonData/data/beampipe/2015/v1/beampipe.xml',
+        'Geometry/CMSCommonData/data/beampipe.xml',
```
Comment (ianna) [inline Geometry/HcalCommonData/python/testPhase0GeometryXML_cfi.py]: @bsunanda - I think, you need 2015 beampipe. The other may have overlaps.

File: Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py [hunk 2]
```diff
@@ -234,8 +234,8 @@
         'Geometry/MuonCommonData/data/rpcf/2015/v1/rpcf.xml',
         'Geometry/MuonCommonData/data/csc/2015/v1/csc.xml',
         'Geometry/MuonCommonData/data/mfshield/2017/v1/mfshield.xml',
-        'Geometry/MuonCommonData/data/PhaseII/gemf.xml',
-        'Geometry/MuonCommonData/data/PhaseII/gem11.xml',
+        'Geometry/MuonCommonData/data/gemf/TDR_BaseLine/gemf.xml',
+        'Geometry/MuonCommonData/data/gem11/Run2/gem11.xml',
```
Comment (ianna) [inline Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py]: @bsunanda - Note, that from now on modifications to gemf.xml or gem11.xml will be rejected. Please, move them to a versioned structure to allow further modifications. For example: gem11/2017/v1/gem11.xml

**Review by ianna — CHANGES_REQUESTED** [review]

**Comment by kpedro88** [comment]
I think this can also cause a conflict with #18373, so whoever is not merged first needs to rebase and rerun the 2023 geometry script

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 Could you sign this please. The next one is Hcal which is a beast

**Comment by kpedro88** [comment]
@bsunanda you need to rebase this on the latest IB and rerun the 2023 geometry script, as I noted before.

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by kpedro88** [comment]
@davidlange6 @bsunanda NB, now this can have a conflict with #18524 - probably easier to merge this one first, then I can update that one

**Comment by kpedro88** [comment]
@bsunanda #18524 is now merged, so this needs to be rebased to the `CMSSW_9_1_X_2017-05-02-1100` IB with the geometry script rerun for the new configurations D15,D16,D17

**Comment by kpedro88** [comment]
-1
until rebased/updated

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@kpedro88 @ianna I have update this PR yet again. Please sign this

**Comment by ianna** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**